zkvm/src/verifier.rs

```diff
@@ -108,7 +101,12 @@ impl<'t> Verifier<'t> {
             verifier.deferred_operations.push(
                 // TODO: change this to use multi-message context
                 tx.signature
-                    .verify(&mut signtx_transcript, key_agg_fn(&verifier.signtx_keys)?)
+                    .verify(
+                        &mut signtx_transcript,
+                        Multikey::new(verifier.signtx_keys)
```

Comment:
Maybe add a comment that we'd need to sign the contract IDs per key (in addition to txid) via multi-message API?

```suggestion
                        // TODO: change this to a proper multi-message API (#176, #266) and use (Contract ID, pubkey) pairs.
                        Multikey::new(verifier.signtx_keys)
```